### DISCUSSION TITLE
REST query bindings values used in Data Provider disappear

### DISCUSSION THREAD

**Discussion by Lanhild** [Budibase/budibase#11306] [Help] [answered]
For an odd and unknown reason, Budibase randomly removed the values I put in my bindings of the Data Provider. It's a Data Provider using a PSQL query, but when clicking on the query settings of the data provider, the value I filled have disappeared. 

This broke half of my application and is a very strange bug.

![image](https://github.com/Budibase/budibase/assets/73610779/820e52a6-5c87-4c4e-8ec2-6b0c23664c65)

I'm self hosting v2.8.10

**Comment by Lanhild** [comment]
for an example see this video
![https://github.com/Budibase/budibase/assets/73610779/b7f5e8ce-5101-4391-860f-cf587d36f0ef](https://github.com/Budibase/budibase/assets/73610779/b7f5e8ce-5101-4391-860f-cf587d36f0ef)

**Comment by Lanhild** [comment]
This seems like a bug that is coming and going - it's not consistent and I don't have any logs related to this :/

Anyhow, very weird behavior

**Comment by melohagan** [comment, marked as answer]
Thanks for raising @Lanhild 

This has been fixed! https://github.com/Budibase/budibase/pull/11338
